R/fix_duplicates.R

```diff
@@ -0,0 +1,80 @@
+#' Fix the West Dunbartonshire duplicates
```

Comment:
I like the separation of this, but could you rename the script and functions to reference homelessness, please? At the moment it's not 100% clear which dataset they refer to from the titles.